pkg/kine/server/get.go

```diff
@@ -23,13 +23,6 @@ func (l *LimitedServer) get(ctx context.Context, r *etcdserverpb.RangeRequest) (
 		attribute.Int64("revision", r.Revision),
 	)
 
-	if len(r.RangeEnd) != 0 {
-		return nil, fmt.Errorf("unexpected rangeEnd: want empty, got %s", r.RangeEnd)
-	}
-	if r.Limit != 0 {
-		return nil, fmt.Errorf("unexpected limit: want 0, got %d", r.Limit)
-	}
```

Comment:
Previous check (currently also in kine), essentially never executed:
```
	if r.Limit != 0 && len(r.RangeEnd) != 0 {
		return nil, fmt.Errorf("invalid combination of rangeEnd and limit, limit should be 0 got %d", r.Limit)
	}
```
As there is a restriction already in limited.go for this get request:
```
	if len(r.RangeEnd) == 0 {
		return l.get(ctx, r)
	}
```
Zero limit means -> no limit for the request
In all cases we want 1 revision return by this request.